web-animations-1/Overview.bs

```diff
@@ -924,6 +931,32 @@ matching condition from below:
 
 </div>
 
+### The current phase of an animation ### {#the-current-phase-of-an-animation}
+
+In addition to their <a>current time</a>, an <a>animation</a> also provides
+their <a>associated effect</a> with a <dfn>current phase</dfn>. This is used in
```

Comment:
Super minor nit: "...an animation also provides their associated effect with..." is a bit odd.

Either:
> ...animations provide their associated effects with...

or:
> ...an animation provides its associated effect with...

would be better. Typically I think we use the former, plural form.

(Incidentally I dropped the "also" since it's redundant with the "In addition")